### DISCUSSION TITLE
Update to German locale [de]

### DISCUSSION THREAD

**Discussion by adriaanb** [aclap-dev/video-downloadhelper#614] [Language & Translation]
Hello,

I've taken some time to complete the missing strings for the German localization.
Please find my suggested translations in the file attached.

Hope it's helpful :)

[de.txt](https://github.com/aclap-dev/video-downloadhelper/files/15436989/de.txt)

**Comment by paulrouget** [comment]
Awesome! Thank you :) will be part of 9.0.1.25! ❤️

**Comment by Xetobal** [comment]
Here are some more translations. Hope it's helpful as well
[de.txt](https://github.com/aclap-dev/video-downloadhelper/files/15495538/de.txt)

**Comment by paulrouget** [comment]
Sorry for the late reply. This will be in v9.1.
